docs/configuration/node-config.md

```diff
@@ -159,6 +159,7 @@ This section contains the configuration options for an indexer. The split store
 | `merge_concurrency` | Maximum number of merge operations that can be executed on the node at one point in time. | `(2 x num threads available) / 3` |
 | `enable_otlp_endpoint` | If true, enables the OpenTelemetry exporter endpoint to ingest logs and traces via the OpenTelemetry Protocol (OTLP). | `false` |
 | `cpu_capacity` | Advisory parameter used by the control plane. The value can expressed be in threads (e.g. `2`) or in term of millicpus (`2000m`). The control plane will attempt to schedule indexing pipelines on the different nodes proportionally to the cpu capacity advertised by the indexer. It is NOT used as a limit. All pipelines will be scheduled regardless of whether the cluster has sufficient capacity or not. The control plane does not attempt to spread the work equally when the load is well below the `cpu_capacity`. Users who need a balanced load on all of their indexer nodes can set the `cpu_capacity` to an arbitrarily low value as long as they keep it proportional to the number of threads available. | `num threads available` |
+| `enable_cooperative_indexing` | Enable to limit the memory used by concurrent indexing workbenches. Useful on workloads with high numbers of actively written indexes (>50) but might decrease the overall indexing throughput. | `false` |
```

Comment:
It's more about sharing ressources efficiently and fairly when the number of indexes is >> than the number of cores. We can just keep your second sentence:
```
Useful for workloads with a high numbers of active indexes (>50) but might decrease the overall indexing throughput.
```